### PR TITLE
Change config items

### PR DESCRIPTION
**Pull request by cfokkens**
#8

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -20,14 +20,14 @@ source-code: https://github.com/CollectionBuilder/collectionbuilder-gh
...
 # tagline, a short phrase that will appear throughout the site in the top banner
-tagline: a template for creating simple digital exhibits
+tagline: Maps of the Development of Vancouver, BC
 # description appears in meta tags and other locations
 # this description might appear in search result lists, keep around 160 characters max
-description: "CollectionBuilder-GH is a template for creating small digital collection exhibits on GitHub Pages designed for teaching digital library skills"
+description: "UBC Maps Collection is a collection of maps of Vancouver available through UBC's library"
 # creator of the digital collection, to appear in meta tags; we typically use our GitHub usernames but feel free to just use your name
-author: CollectionBuilder
+author: UBC GIS Library
```
Comment (ect123) [inline _config.yml]: Hi Charlie, can you jump in this branch and call it "UBC Library GIS" instead? Thanks!

File: _config.yml [hunk 2]
```diff
@@ -20,14 +20,14 @@ source-code: https://github.com/CollectionBuilder/collectionbuilder-gh
...
 # title of site appears in banner
-title: CollectionBuilder-GH 
+title: UBC Maps Collection
 # tagline, a short phrase that will appear throughout the site in the top banner
-tagline: a template for creating simple digital exhibits
+tagline: Maps of the Development of Vancouver, BC
 # description appears in meta tags and other locations
 # this description might appear in search result lists, keep around 160 characters max
-description: "CollectionBuilder-GH is a template for creating small digital collection exhibits on GitHub Pages designed for teaching digital library skills"
+description: "UBC Maps Collection is a collection of maps of Vancouver available through UBC's library"
```
Comment (ect123) [inline _config.yml]: Could probably say "through UBC's Koerner Library"

**Review by ect123 — COMMENTED** [review]
Just a couple small things to fix - otherwise GREAT!

**Comment by cfokkens** [comment]
I have fixed these items and pushed them to this branch, but I'm not sure if I need to do anything to add them to this pull request!

*Nevermind, I see there is a link to my changes above

**Pull request merged**